docs/topics/develop/patterns/_category_.json

```diff
@@ -0,0 +1,9 @@
+{
+  "label": "Patterns",
```

Comment:
I'd like to get @FractalJackal's opinion on whether we should be calling these `Patterns` or `How To`